src/main/flight/mixer.c

```diff
@@ -426,6 +435,20 @@ static void applyMixToMotors(const float motorMix[MAX_SUPPORTED_MOTORS], motorMi
             motor[i] = motor_disarmed[i];
         }
     }
+
+#ifdef USE_WING
+    float averageMotor = 0.0f;
+    if (motorIsEnabled()) {
+        for (int i = 0; i < mixerRuntime.motorCount; i++) {
+            if (motorIsMotorEnabled(i)) {
+                averageMotor += (motorConvertToExternal(motor[i]) - PWM_RANGE_MIN) / (float)PWM_RANGE;
+            }
+        }
+        averageMotor /= mixerRuntime.motorCount;
```

Comment:
for advanced model we need to know both throttle^2 and throttle. And in case of multiple motors to be accurate we need to know throttle^2 and throttle for each of the motors individually.

So i guess i will have to get rid of just average motor output function, and do the math on motors in PIDloop?

Reply:
You can handle BASIC by summing thrust^2 and than taking sqrt of result.

For dual propeller, you can compute total force (prom propeller model; using actual speed (pitch speed ratio). But in this situation, it quite possible that one propeller will have pitch speed considerably lower that plane speed. And currently used model does not work very well in this mode.

IMO we can ignore this problem in this PR. And in subsequent improvements, I'd try to improve model used (there is a lot of wind turbine theory available. so it should be relatively easy) and then fix/improve this.

Reply:
@ledvinap i am a little lost here, could you plz help. Because we also want to adjust motor outputs with the voltage sag by a factor of `getBatteryVoltageLatest() / 100.0f / pidRuntime.tpaSpeed.maxVoltage`.